### ISSUE TITLE
with 0.5.0 it seems flatpak is now a hard dependency

### ISSUE THREAD

**Issue by philmmanjaro** [vinifmor/fpakman#36]
Currently I get the following when **flatpak** not installed is. Maybe **snapd** is also a hard dependency now.

```
qt5ct: using qt5ct plugin
qt5ct: D-Bus system tray: no
/bin/sh: flatpak: command not found
Traceback (most recent call last):
  File "/usr/bin/fpakman", line 11, in <module>
    load_entry_point('fpakman==0.5.0', 'console_scripts', 'fpakman')()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2843, in load_entry_point
    return ep.load()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2434, in load
    return self.resolve()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2440, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3.7/site-packages/fpakman/app.py", line 145, in <module>
    warnings = manager.list_warnings()
  File "/usr/lib/python3.7/site-packages/fpakman/core/controller.py", line 283, in list_warnings
    man_warnings = man.list_warnings()
  File "/usr/lib/python3.7/site-packages/fpakman/core/flatpak/controller.py", line 184, in list_warnings
    if not flatpak.has_remotes_set():
  File "/usr/lib/python3.7/site-packages/fpakman/core/flatpak/flatpak.py", line 245, in has_remotes_set
    return bool(system.run_cmd('{} remotes'.format(BASE_CMD)).strip())
AttributeError: 'NoneType' object has no attribute 'strip'
```

**Comment by vinifmor** [comment]
@philmmanjaro, I already published a fix to the **staging** as commented in Manjaro Forum. I've already checked for **snapd**, and it's working. Please, give me a feedback if it's working without **flatpak** now. Thanks again for the feedback.

**Comment by fhdk** [comment]
I just tested today.

I didn't have flatpak installed - and no flatpakw were listed.

Great app :+1:

**Comment by vinifmor** [comment]
Fixed in **0.5.1**
